time.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"golang.org/x/sys/unix"
+)
+
+const agentStartedEvent = "agent_started"
+
+//Event that will be reported for time tracing
+type eventTime struct {
+	Event string
+	Time  unix.Timespec
```

Comment:
Hi @jcvenegas You asked me what time format I thought we should print in. My gut feeling is that we should probably use the same format as the other logs we produce within CC - so, for instance, it looks like the runtime logs use the logrus internal Time data:
https://github.com/clearcontainers/runtime/blob/9be2f54966b90014251afa5cebdf1ee36a85503f/logger.go#L137
If we can, I think we should do the same - what do you think @jodh-intel